king_phisher/client/application.py

```diff
@@ -552,6 +554,13 @@ def do_config_load(self, load_defaults):
 			for key, value in client_template.items():
 				if not key in self.config:
 					self.config[key] = value
+			env_proxy = os.environ['HTTPS_PROXY']
```

Comment:
This will raise a `KeyError` if `HTTPS_PROXY` is not set. Instead use `os.environ.get('HTTPS_PROXY')`.